debvader/normalize.py

```diff
@@ -1,8 +1,32 @@
-def linear_normalization_cosmos(x):
+import numpy as np
+
+
+def linear_normalization_cosmos(x, direction="normalize"):
```

Comment:
instead of choosing the direction through arguments I would rather create two normalization functions.
Call them the way you want (forward-backward seems the best option to me).
But this will be much simpler, especially if you plan on feeding the normalization functions to other pieces of code (it's much harder to also feed the function arguments with it)